servicecontrol/plugins.md

```diff
@@ -124,16 +136,19 @@ However, depending on the saga's update frequency, it may also result in a large
 
 ### ServiceControl DebugSession Plugin
```

Comment:
I think we should keep the names the same as the nuget names to avoid confusion.